releases/release-1.32/release-notes/maps/pr-124285-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124285
+releasenote:
+  text: 'CRI: Add field to support CPU affinity on Windows'
```

Can you test out the `krel release-notes validate --path-to-release-notes ~/kubernetes/sig-release/releases/release-1.32/release-notes` command to see if it would have caught the missing period? 

```suggestion
  text: 'CRI: Add field to support CPU affinity on Windows.'
```